### PR TITLE
Update Membership UK loyalty message

### PR DESCRIPTION
**Pull request by davidrapson**
- Move the UK membership message out of the AB tests code into a real module as it's really not a test anymore.
- Update the message to be a thank-you message rather than a quote.

screen shot 2015-10-14 at 11 48 58

Ideally I'd also like to exclude the message for existing Supporters at this point, @jbreckmckye @Calanthe are you still waiting on something from us (Membership) to do that within frontend?

### CODE REVIEW

**Comment by Calanthe** [comment]
We are still discussing it with @ostapneko

**Comment by davidrapson** [comment]
@Calanthe Yeah, spotted that as soon as I wrote that. We can cope without it for this change for now I think, if you've got time to review this?

File: static/src/javascripts/projects/common/modules/commercial/membership-messages.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    function canRun() {
+        /**
+         * - Exclude adblock users to avoid conflicts with similar adblock Supporter message
+         * - Exclude mobile/small-screen devices
+         * - Only show for UK edition
+         * - Only show on Article pages
+         * - Only show to visitors who have viewed more than 10 pages.
+         */
+        var alreadyVisited = storage.local.get('gu.alreadyVisited') || 0;
+        return config.switches.membershipMessages &&
```
Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/membership-messages.js]: Maybe we should start using the functionality from the https://github.com/guardian/frontend/pull/10653 here?

**Comment by davidrapson** [comment]
@Calanthe Moved the rules that made sense into the commercial policies, let me know if it looks OK?

File: static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js [hunk 2]
```diff
@@ -53,6 +55,17 @@ define([
         }
     };
 
+    policies.membershipMessages = function () {
+        if (!detect.adblockInUse() &&
+            detect.getBreakpoint() !== 'mobile' &&
+            config.page.contentType === 'Article'
+        ) {
```
Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js]: Can you also move `config.page.edition === 'UK'` here?

Comment (davidrapson) [inline static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js]: I'd like to move the US version of this message into `membership-messages` too, so the `UK` check is fairly circumstantial. I've just moved the shared rules into here.

Comment (Calanthe) [inline static/src/javascripts/projects/common/modules/commercial/commercial-feature-policies.js]: ok

**Comment by Calanthe** [comment]
:+1:

File: static/src/javascripts/projects/common/views/membership-message.html [hunk 3]
```diff
@@ -4,6 +4,6 @@
 <ul class="site-message__actions u-unstyled">
     <li class="site-message__actions__item">
         <%=arrowWhiteRight%>
-        <a href="<%=supporterLink%>" target="_blank" data-link-name="read more"><%=linkText%></a>
+        <a href="<%=linkHref%>" target="_blank" data-link-name="read more"><%=linkText%></a>
```
Comment (rtyley) [inline static/src/javascripts/projects/common/views/membership-message.html]: Changed here, but not in the parameters that point to the template...

**Pull request merged**